contrib/for-tests/mount-tester/mt.go

```diff
@@ -42,6 +46,9 @@ func init() {
 	flag.StringVar(&newFilePath0644, "new_file_0644", "", "Path to write to and read from with perm 0644")
 	flag.StringVar(&newFilePath0666, "new_file_0666", "", "Path to write to and read from with perm 0666")
 	flag.StringVar(&newFilePath0777, "new_file_0777", "", "Path to write to and read from with perm 0777")
+	flag.StringVar(&readWriteNewFilePath, "rw_new_file", "", "Path to write to and read from")
+	flag.StringVar(&readFileInLoopContentPath, "file_content_in_loop", "", "Path to read the file content in loop from")
```

Comment:
@dalanlan The `readWriteNewFilePath` variable is never used to drive creating the new file.  Perhaps you lost something in your rebase?  I'm not sure that we need this now that there are other options for creating new files with different permissions.